### ISSUE TITLE
0.8.0.1 build error (in stackage nightly)

### ISSUE THREAD

**Issue by bergmark** [GregorySchwartz/diversity#3]
Several errors like this one:

```
src/src-exec/Main.hs:268:6: error:
    • Variable not in scope:
        (<>) :: InfoMod a72 -> InfoMod a73 -> InfoMod Options
    • Perhaps you meant one of these:
        ‘<$>’ (imported from Options.Applicative),
        ‘<*>’ (imported from Options.Applicative),
        ‘*>’ (imported from Options.Applicative)
```

**Comment by bergmark** [comment]
I noticed I shouldn't downgrade diversity, 0.8.0.1 was already in nightly. Did some dependency stop re-exporting `(<>)`?

PS. 0.8.0.0 failed to build with

```
Building diversity-0.8.0.0...
Preprocessing library diversity-0.8.0.0...
[1 of 6] Compiling Math.Diversity.Statistics ( src/src-lib/Math/Diversity/Statistics.hs, dist/build/Math/Diversity/Statistics.o )
[2 of 6] Compiling Math.Diversity.RandomSampling ( src/src-lib/Math/Diversity/RandomSampling.hs, dist/build/Math/Diversity/RandomSampling.o )
[3 of 6] Compiling Math.Diversity.Types ( src/src-lib/Math/Diversity/Types.hs, dist/build/Math/Diversity/Types.o )
[4 of 6] Compiling Math.Diversity.Diversity ( src/src-lib/Math/Diversity/Diversity.hs, dist/build/Math/Diversity/Diversity.o )
[5 of 6] Compiling Math.Diversity.GenerateDiversity ( src/src-lib/Math/Diversity/GenerateDiversity.hs, dist/build/Math/Diversity/GenerateDiversity.o )

src/src-lib/Math/Diversity/GenerateDiversity.hs:32:21: error:
    Ambiguous occurrence ‘getField’
    It could refer to either ‘Data.Fasta.String.getField’,
                             imported from ‘Data.Fasta.String’ at src/src-lib/Math/Diversity/GenerateDiversity.hs:16:1-24
                             (and originally defined in ‘Data.Fasta.String.Utility’)
                          or ‘Math.Diversity.GenerateDiversity.getField’,
                             defined at src/src-lib/Math/Diversity/GenerateDiversity.hs:27:1

src/src-lib/Math/Diversity/GenerateDiversity.hs:73:28: error:
    Ambiguous occurrence ‘getField’
    It could refer to either ‘Data.Fasta.String.getField’,
                             imported from ‘Data.Fasta.String’ at src/src-lib/Math/Diversity/GenerateDiversity.hs:16:1-24
                             (and originally defined in ‘Data.Fasta.String.Utility’)
                          or ‘Math.Diversity.GenerateDiversity.getField’,
                             defined at src/src-lib/Math/Diversity/GenerateDiversity.hs:27:1
```

**Comment by GregorySchwartz** [comment]
Yes, optparse-applicative changed their api to use semigroups. I've been fixing these issues as they've been coming, but the discrepency between lts and nightly means I can only support nightly right now. Resolved (0.8.0.2 for nightly).

P.S. Thanks, I probably forgot to deprecate that version.
